frontend/src/dispatcher/js-messages.ts

```diff
@@ -39,6 +39,8 @@ export class FrontendDocumentDetails extends DocumentDetails {
 	readonly id!: BigInt;
 }
 
+export type MouseCursorIcon = string;
```

Comment:
Change the type from `string` to `"default" | "zoom-in" | "zoom-out" | "grabbing" | "crosshair"`

Reply:
And please move this down next to where it's used. Right above its first usage (in the `Transform(...)` would be best.